omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -100,6 +100,7 @@
             move "#{install_dir}/etc/datadog-agent/conf.d", "/etc/datadog-agent", :force=>true
             unless heroku_target?
               move "#{install_dir}/etc/datadog-agent/system-probe.yaml.example", "/etc/datadog-agent"
+              copy "/etc/datadog-agent/system-probe.yaml.example", "/etc/datadog-agent/system-probe.yaml"
```

Comment:
Probably missing some `#{install_dir}` here

Reply:
Not necessarily  since the file is copied just before in that folder, however this will override any custom config the customer had there.
You can't conditionally copy a file to the user's folders from the omnibus recipes, you'll have to use the package scripts in `omnibus/package-scripts/`

Reply:
Thanks for the info! I see a few scripts inside `omnibus/package-scripts/`, do you know which one is the most appropriate for this?

Reply:
Thanks for jumping in @chouquette, I didn't think about that aspect yesterday 🙇.

@rarguelloF I'd use the post install scripts to check if the probe config exists, and copy the example over if it doesn't.